samples/bluetooth/central_hr/prj_minimal.conf

```diff
@@ -101,3 +101,14 @@ CONFIG_BT_L2CAP_TX_BUF_COUNT=2
 CONFIG_BT_CTLR_RX_BUFFERS=1
 CONFIG_BT_BUF_ACL_TX_COUNT=3
 CONFIG_BT_BUF_ACL_TX_SIZE=27
+
+# Limit the number of key slots in PSA Crypto core to reduce
+# RAM footprint
+CONFIG_MBEDTLS_PSA_KEY_SLOT_COUNT=5
+
+# This is required because otherwise there won't be
+# __heap_start/__heap_end symbols defined in the build. This
+# is due to Mbed TLS which at boot uses standard calloc/free
+# functions and only after the initialization done from Zephyr
+# it switches to its internal memory management functions.
+CONFIG_COMMON_LIBC_MALLOC=y
```

Comment:
Will this be fixed eventually or we will always have to enable this option?

Reply:
I tried to nicely fix this on Mbed TLS side (as part of #80136 - [this ](https://github.com/zephyrproject-rtos/zephyr/commit/9dc4b337b75a1581b80fab74d2d155a712205a16)was the fix that I tried in case you are curious to check) but unfortunately other failures appeared in that PR. Since we're a bit in a rush here I decided to keep this dirty fix, add a long comment to explain, and fix it later with a follow-up PR.